desk/app/channels-server.hoon

```diff
@@ -3,7 +3,7 @@
 ::    this is the server-side from which /app/channels gets its data.
 ::
 /-  c=channels, g=groups
-/+  utils=channel-utils, imp=import-aid
+/+  utils=channel-utils, imp=import-aid, em=emojimart
 /+  default-agent, verb, dbug, neg=negotiate
 ::
 %-  %-  agent:neg
```

Comment:
Since we're changing the protocol, you need to bump the version here, to `~.channels^%2`.

Then in the channels agent, bump the _expected_ number for channels-server agents to match.